home-data/src/main/java/com/softserveinc/ita/homeproject/homedata/entity/Cooperation.java

```diff
@@ -0,0 +1,22 @@
+package com.softserveinc.ita.homeproject.homedata.entity;
+
+import java.io.Serializable;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.OneToMany;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "cooperation")
```

Comment:
As we agreed on our decision that we use plural names for our DB table names let's follow it.